### ISSUE TITLE
Add timezone settings to printers

### ISSUE THREAD

**Issue by davidcorry** [nordprojects/sirius#5]
I tried tracking down if I can change this setting myself, but it looks like it's all backend stuff - it would be great if I could set the timezone of my printer to my local timezone, rather than having it be GMT. That way, the date and time header at the top of my print outs would make a little more sense.

I might get around to creating my own pull request on this, but I thought I'd throw this out there first in case anyone else wanted to tackle it first.

(Thanks for helping me revive my Little Printer, by the way!)

**Comment by joerick** [comment]
Agreed, a little setting in the backend for timezone would be great. PRs welcome!
